docs/guides/index.md

```
@@ -27,6 +27,12 @@ Structure larger applications effectively:
 - Composing functionalities
 - Managing dependencies between packages
 
+### [Configuration Management](configuration.md)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)